ffi/wgpu.h

```diff
@@ -994,6 +996,10 @@ void wgpu_request_adapter_async(const WGPURequestAdapterOptions *desc,
 
 void wgpu_sampler_destroy(WGPUSamplerId sampler_id);
 
+void wgpu_set_log_callback(WGPULogCallback callback);
+
+void wgpu_set_log_level(int level);
```

Comment:
Consider changing `int` to an enum type here like this:
```c
typedef enum {
  WGPULogLevel_DEBUG = 0,
  WGPULogLevel_INFO = 1,
  // etc.
} WGPULogLevel

void wgpu_set_log_level(WGPULogLevel level);
```

Reply:
Makes sense, I think. Would it make sense to copy over the `log::Level` enum, or perhaps a subset?

Reply:
copying the enum seems fine to me, if it's not already repr(C)